mc/Mongo-Core.package/MongoDatabase.class/instance/isMaster.st

```diff
@@ -0,0 +1,7 @@
+operations
+isMaster
+
+	| reply |
+	reply := self command: (OrderedDictionary new at: #ismaster put: 1; yourself).
```

Comment:
What's the reason to use OrderedIdentityDictionary in many other places?

Reply:
Frankly I don't know, a reason may be to allow repeated String keys but I don't know in which special cases. I think we should unify how the "JSON dictionaries" (Dictionaries that will be converted to json in a mongo call) are created.

Reply:
I created issue #61 to maybe discuss more about this.